src/LoginScreen.js

```diff
@@ -19,7 +19,7 @@ export default function LoginScreen (){
     setPlayerEmail(email)
     changeIsLoggedIn(true);
     let name = response.profileObj.name;
-    socket.emit('login', {name, email});
+    socket.emit('login', {name, email, room:''});
```

Comment:
Do we need to pass a value for `room` here?

Reply:
no i will take it out